multiqc/utils/search_patterns.yaml

```diff
@@ -876,3 +876,6 @@ verifybamid/selfsm:
 whatshap/stats:
   contents: "#sample	chromosome	file_name	variants	phased	unphased	singletons"
   num_lines: 1
+mosaicatcher:
+  contents: "# sample:  Sample (has multiple cells)"
```

Comment:
Is it guaranteed that the value of "sample" is always going to be "Sample (has multiple cells)"?

Reply:
Ah, checked the pipeline - seems like the header is a constant